Library/Homebrew/cmd/update-report.rb

```diff
@@ -302,16 +303,11 @@ def migrate_legacy_repository_if_necessary
     $stderr.puts e.backtrace
   end
 
-  def link_completions_and_docs(repository = HOMEBREW_REPOSITORY)
+  def link_completions_manpages_and_docs(repository = HOMEBREW_REPOSITORY)
     command = "brew update"
-    link_src_dst_dirs(repository/"completions/bash",
-                      HOMEBREW_PREFIX/"etc/bash_completion.d", command)
-    link_src_dst_dirs(repository/"docs",
-                      HOMEBREW_PREFIX/"share/doc/homebrew", command, link_dir: true)
-    link_src_dst_dirs(repository/"completions/zsh",
-                      HOMEBREW_PREFIX/"share/zsh/site-functions", command)
-    link_src_dst_dirs(repository/"manpages",
-                      HOMEBREW_PREFIX/"share/man/man1", command)
+    link_completions(repository, command)
+    link_manpages(repository, command)
+    link_docs(repository, command)
```

Comment:
I'm not sure that I understand your question. Are you talking about `link_completions_manpages_and_docs()` or link_docs() method here?
If it's the former, the method exists to DRY up the same code in the taps.
If it's the latter, the method exists for consistency with the other `link_` methods.

Or perhaps you're suggesting to inline the calls to `link_*` at the call sites? I'm a fan of smaller methods rather than larger generally, and we need to call `Tap.link_completions_and_manpages` externally so I think for consistency this means this stays.